share/availability/light/availability.go

```diff
@@ -97,7 +97,6 @@ func (la *ShareAvailability) SharesAvailable(ctx context.Context, header *header
 	}
 
 	if err := dah.ValidateBasic(); err != nil {
-		log.Errorw("DAH validation failed", "error", err)
 		return err
 	}
```

Comment:
Btw, we may want to remove this check entirely. Headers are guaranteed to be valid at this point.

Reply:
Removed in light simplification PR: https://github.com/celestiaorg/celestia-node/pull/3895